prefect_bitbucket/credentials.py

```diff
@@ -7,20 +7,23 @@
 
 
 class BitBucketCredentials(Block):
-    """
-    Store a BitBucket access token to interact with private BitBucket
-    repositories.
+    """Store BitBucket credentials to interact with private BitBucket repositories.
 
-    Attributes:
+    Attributes
+    ----------
```

Comment:
Really thank you for trying out pydocstyle! Can we disable the pydocstyle code that requires this; I think I prefer the original `Attributes:`